libexec/image-handler.sh

```diff
@@ -80,10 +80,12 @@ case "$SINGULARITY_IMAGE" in
 
         eval_abort "$SINGULARITY_libexecdir/singularity/python/import.py"
 
+         
         for i in `cat "$SINGULARITY_CONTENTS"`; do
             name=`basename "$i"`
             message 1 "Exploding layer: $name\n"
             zcat "$i" | (cd "$SINGULARITY_ROOTFS"; tar --exclude=dev/* -xf -) || exit $?
+            chmod -R +w "$SINGULARITY_ROOTFS"
```

Comment:
I've added the message for a check if this part of the code is actually running.